tests/benchdnn/inputs/ip/test_ip_acl

```diff
@@ -24,3 +24,10 @@
 --wtag=abx,axb
 # 2d-spatial dimensions
 --batch=shapes_googlenet_v1
+
+# Tests for external blocked weights layout.
```

Comment:
Thanks for adding these examples. Just to check, does this pass on Graviton 2, 3 (maybe even 4?)? The chosen (and therefore supported) blocked weight format depends on the micro architecture, so it might be worth putting this in a machine specific input file.

Reply:
good point. but it will just skip on the unsupported platforms right. could you please elaborate what you mean by machine specific input file. do you mean adding a new input file something like `test_ip_acl_neoversev1` ?
Sorry, I'm extending the benchdnn unit tests for the first time, might be missing some basics.

Reply:
currently the test is passing on Graviton3 and is skipped on Graviton2.

Reply:
Yes, I meant something like that, but I forgot how the skip mechanism works. That should work fine, thanks!